polyfills/document/elementsFromPoint/config.toml

```
@@ -0,0 +1,67 @@
+#### Aliases
+# An alias is another name for the polyfill that can be used when requesting a bundle.
+# For example, the alias `es5` is used to replicate an ECMAScript 5 environment.
+aliases = []
+
+#### Dependencies
+# List any web platform features that you use in your polyfill.
+# For example:
+# dependencies = [
+#    "Object.defineProperty"
+# ]
+dependencies = []
```

The polyfill uses Array.prototype.map on line 24 https://github.com/Financial-Times/polyfill-library/pull/1038/files#diff-0726f42f831bc5f6492c044eeffddf5277c06df20b11b41c5c47581740214c51R24

We need to include that as a dependency for browsers which don't have the feature
```suggestion
dependencies = [
	"Array.prototype.map"
]
```

---

added 86efe2b1656a1e0fea3e93a60106118272311265